nix/nixos/cardano-node-service.nix

```diff
@@ -20,6 +20,10 @@ let
         TargetNumberOfKnownPeers = cfg.targetNumberOfKnownPeers;
         TargetNumberOfEstablishedPeers = cfg.targetNumberOfEstablishedPeers;
         TargetNumberOfActivePeers = cfg.targetNumberOfActivePeers;
+        TestEnableDevelopmentNetworkProtocols = true;
+        MaxConcurrencyBulkSync = 2;
+        TraceBlockFetchClient = true;
+        TraceChainSyncClient = true;
```

Comment:
I think you should add a comment that these options are only for p2p testnet. If it turns out to be rather costly they should be disabled in the future.